### DISCUSSION TITLE
Need help figuring out the font for Tokyo theme

### DISCUSSION THREAD

**Discussion by poqdavid** [JanDeDobbeleer/oh-my-posh#3163] [Troubleshoot] [answered]
@x-daht I have been trying to get this to look like the image but I can't, anyone knows what font is used on this image?

![tokyo-905097f8b02151ca72c2534451a69816](https://user-images.githubusercontent.com/9592210/205466056-66d43a1d-93c5-4aa1-8fc5-74f1b096435e.png)

@JanDeDobbeleer  Also wanted to say as a suggestion the theme list should include font and font size that be amazing

**Comment by lewis-yeung** [comment, marked as answer]
It's [Hack Nerd Font](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.2.2/Hack.zip).

**Reply by JanDeDobbeleer** [reply]
> Also wanted to say as a suggestion the theme list should include font and font size that be amazing

This is always Hack as stated by @lewis-yeung. The size isn't relevant as that's something that differs from terminal to terminal (how it renders).

**Reply by poqdavid** [reply]
Oh, it felt like slightly different tyvm
